### PR TITLE
Adds Travis-CI support

### PR DESCRIPTION
**Pull request by zimbatm**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+# use faster container
+sudo: false
+language: java
+jdk:
+  - oraclejdk8
+  - oraclejdk7
+  - openjdk6
+  - openjdk7
+# Fixes GPG dependency issues
+install: mvn install -DskipTests=true -Dgpg.skip=true
```
Comment (mdpye) [inline .travis.yml]: Do we need to install it? Can travis not just run `mvn test`?

Comment (zimbatm) [inline .travis.yml]: The install phase is probably cached ?

Comment (zimbatm) [inline .travis.yml]: The other solution I saw was to put this dependency into a separate `<profile>`

Comment (zimbatm) [inline .travis.yml]: http://stackoverflow.com/questions/14825039/suppressing-gpg-signing-for-maven-based-continous-integration-builds-travis-ci

**Pull request merged**